config-generators/dab-config.sql.reference.json

```diff
@@ -946,6 +946,39 @@
           "actions": [ "*" ]
         }
       ]
+    },
+    "Bookmarks": {
```

Comment:
Given the checked in config files in the Test project, we may remove these reference files altogether

Reply:
ah gotcha.

Reply:
since this PR does not have removing config files in scope, will save for separate PR. So will update existing config files until removed.

Reply:
i ended up removing the three reference configs.